src/services/settings.ts

```diff
@@ -53,6 +53,38 @@ class ComposeAreaSettings {
     }
 }
 
+class UserInterfaceSettings {
+    private readonly settingsService: SettingsService;
+
+    constructor(settingsService: SettingsService) {
+        this.settingsService = settingsService;
+    }
+
+    public getUserInterface(): threema.UserInterface {
+        return this.parseUserInterface(this.settingsService.retrieveUntrustedKeyValuePair('userInterface', false));
+    }
+
+    public setUserInterface(userInterface: string | threema.UserInterface): void {
+        this.settingsService.storeUntrustedKeyValuePair('userInterface', this.parseUserInterface(userInterface).toString());
+    }
+
+    private parseUserInterface(userInterface: any): threema.UserInterface {
+        try {
+            userInterface = parseInt(userInterface, 10);
+        } catch {
+            // Ignored
+        }
```

Comment:
Wouldn't it be better to just treat the user interface variant as a string, instead of converting it to an integer (and then back to a string for storage)? This would also make the template radio button values more readable. (As far as I know, TypeScript const enums can contain strings as well.)

Reply:
I agree, this would be much better :) I changed it. I did this initialy with 0 and 1 because the other settings do it like this as well. But directly with the string is much better :)

Reply:
@dbrgn Just an FYI here. The reason why I chose an integer for newer settings is this: Local storage persists across versions of Threema Web and thus works similarly to a protocol. It's easy to change the name of a mapped item like an integer. But a string ties you to that string. If it's misspelled, it stays misspelled. Same if the semantics change (even if just slightly) or are extended/specialised. One can work around this mishap in the application code by mapping it using an `enum`, but a string enum in TypeScript is IMO unnecessarily clunky compared to `type Type = 'foo' | 'bar'`.

But, granted, the whole thing was hacked in by me very quickly, so it looks ugly in the radio buttons. It doesn't need to be that way, though.